docs_user/modules/proc_migrating-mon-and-mgr-from-controller-nodes.adoc

```diff
@@ -271,17 +239,18 @@ On the oc0-ceph-0:
 8: vlan12    inet 172.16.12.46/24 brd 172.16.12.255 scope global vlan12\       valid_lft forever preferred_lft forever
 ----
 
-Poweroff oc0-controller-1.
-
-Add the new mon on oc0-ceph-0 using the old IP address:
+. Power off oc0-controller-1.
+//kgilliga: Is this something that customers just know how to do?
```

Comment:
Yes, and I think this is an optional step. We are migrating to the new nodes, but we're not suggesting steps to decommission the previous HW.
We can turn this step into a Note I suppose.